http/swagger.yml

```diff
@@ -3365,7 +3368,10 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
-  /query:
+  /api/v2/query:
+    servers:
+      - url: /
+        description: V1 compatible api endpoints.
```

Comment:
This description isn't correct and probably unnecessary?

Reply:
regarding the servers object and rename of the original paths, yes, it's to prevent conflicts while still preserving compatibility.. though judging by the tests, i'm not sure compatibility has been preserved as the client is missing functions.

Reply:
Heh – right, I was just wondering the same thing

Reply:
tests failed due to a limited amount of supported content-types https://github.com/influxdata/oats/issues/16